src/scripts/public/sites/cleanbc/patterns/landing-banners.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Takeover navigation manipulation.
+ */
+const domReady = () => {
+	/*
+	 * SafarIE bug requires 0ms timeout.
+	 */
+	setTimeout(function() {
+		const isHomeBanner = document.querySelectorAll('.home-cover-banner');
+		const isLandingBanner = document.querySelectorAll(
+			'.landing-cover-banner'
+		);
+
+		if (isHomeBanner.length || isLandingBanner.length) {
+			const svg = document.getElementsByTagName('svg')[0];
```

Comment:
Not an array but a nodeList. In a perfect world there would only be one banner per page but I'm making no assumptions on how users will implement layout, so rather than test for querySelector which returns the element or null I use queryselectorAll and get a nodeList which can be examined like an array and will return a length property of 0 when no matches are found.